src/components/Header/Nav/ProductNavigation/Mobile/ProductNavigation.tsx

```diff
@@ -111,23 +92,57 @@ export function ProductNavigation({ productsNav, path }: Props) {
               }}
             >
               <div className={clsx(styles.content, styles[productsSlidePosition])}>
-                <ul className={clsx(styles.productContent)}>
-                  <ProductContent onProductClick={onProductClick} productsNav={productsNav} />
-                </ul>
-                <div className={clsx(styles.subProductContent)}>
-                  <SubProductContent
-                    subProducts={subProducts}
-                    onSubproductClick={onSubproductClick}
-                    currentPath={path}
-                  />
+                <div>
+                  <div className={styles.header}>
+                    <img
+                      alt="Documentation Home"
+                      title="Documentation Home"
+                      style={{ display: "flex" }}
+                      src="/chainlink-docs.svg"
+                      height={30}
+                    />
+                    <Close ref={closeButtonRef} className={styles.closeButton}>
+                      <img src="/assets/icons/close.svg" />
+                    </Close>
+                  </div>
+                  <ul className={clsx(styles.productContent)}>
+                    <button
+                      className={styles.productContentLink}
+                      onClick={() => setProductsSlidePosition("submenu")}
+                      data-testid="sub-product-navigation-trigger-mobile"
+                    >
+                      Resources
+                      <CaretRightIcon />
+                    </button>
+                    <a href="" className={styles.productContentLink}>
+                      Docs
+                    </a>
+                    <a href="" className={styles.productContentLink}>
+                      Demos
+                    </a>
+                    <a href="" className={styles.productContentLink}>
+                      Tools
+                    </a>
+                    <a href="" className={styles.productContentLink}>
+                      ChainLog
+                    </a>
+                  </ul>
+                </div>
+                <div className={clsx(styles.subProductContentPage)}>
+                  <div className={styles.header}>
+                    <button key="back" className={styles.back} onClick={() => setProductsSlidePosition("main")}>
+                      <BackArrowIcon />
+                    </button>
+                    <span className={styles.subProductContentTitle}>Resources</span>
+                    <span></span> {/* Spacer */}
+                  </div>
+                  <div className={styles.subProductContent}>
+                    <MegaMenu />
```

Comment:
For some reason, the megamenu is not going away when I hover on other menu items or even around it. Maybe this is to do with margin or padding that are making its area bigger than it is.

Reply:
I have fixed that by the closing event around it